### PR TITLE
Update to swift 5.1.2

### PR DESCRIPTION
**Pull request by davidrynn**
- Updated via tool
- Updated hash protocol

### CODE REVIEW

**Review by haidalinda — APPROVED** [review]
Haven't reviewed closely - trusting you've tested for your needs

**Review by livtay — APPROVED** [review]

File: Example/Marker-Example.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -468,7 +520,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				INFOPLIST_FILE = "Marker-Example-iOS/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```
Comment (MichaelCampbell) [inline Example/Marker-Example.xcodeproj/project.pbxproj]: let's not change this

File: Example/Marker-Example.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -521,14 +573,14 @@
...
```
Comment (MichaelCampbell) [inline Example/Marker-Example.xcodeproj/project.pbxproj]: same here, let's restore the original value

File: Marker/Marker/Parser/Rule.swift [hunk 3]
```diff
@@ -51,9 +51,8 @@ extension Rule: Equatable {
 // MARK: Hashable
 
 extension Rule: Hashable {
-
-    var hashValue: Int {
-        return openingSymbol.hashValue ^ closingSymbol.hashValue
+    func hash(into hasher: inout Hasher) {
```
Comment (MichaelCampbell) [inline Marker/Marker/Parser/Rule.swift:55]:
```suggestion

    func hash(into hasher: inout Hasher) {
```

**Review by MichaelCampbell — APPROVED** [review]
Looks good, please address the inline comments

**Pull request merged**